views/index.pug

```diff
@@ -0,0 +1,19 @@
+html 
+    body 
+      h1 KIM'S CONVENIENCE
+  h2 Developed By
```

Comment:
The error you're getting is because of indenting. When you indent in Pug you make the element a child of the one above it. Here your h2 is outside of the body.